README.md

```diff
@@ -1420,6 +1422,71 @@ expect(deleteButton).not.toHaveDescription()
 expect(deleteButton).toHaveDescription('') // Missing or empty description always becomes a blank string
 ```
 
+<hr />
+
+### `toHaveSelection`
+
+This allows to assert that an element has a
+[text selection](https://developer.mozilla.org/en-US/docs/Web/API/Selection).
+
+This is useful to check if text or part of the text is selected within an
+element. The element can be either an input of type text, a textarea, or any
+other element that contains text, such as a paragraph, span, div etc.
+
+NOTE: the expected selection is a string, it does not allow to check for
+selection range indeces.
+
+```typescript
+toHaveSelection(expectedSelection: string)
```

Comment:
This does not document the use case where the argument can be omitted to test if there's a selection or not. Although you reflect that in the examples below, in this very markdown file, but still, it'd be good to reflect that here somehow.

```suggestion
toHaveSelection(expectedSelection?: string)
```